Dockerfile

```diff
@@ -1,72 +1,33 @@
-
-# Want to help us make this template better? Share your feedback here: https://forms.gle/ybq9Krt8jtBL3iCk7
-
-ARG NODE_VERSION=20
-
-################################################################################
-# Use node image for base image for all stages.
-FROM node:${NODE_VERSION}-alpine as base
-
-# Set working directory for all build stages.
 WORKDIR /app
 
+COPY package*.json ./
 
-################################################################################
-# Create a stage for installing production dependecies.
-FROM base as deps
-
-# Download dependencies as a separate step to take advantage of Docker's caching.
-# Leverage a cache mount to /root/.npm to speed up subsequent builds.
-# Leverage bind mounts to package.json and package-lock.json to avoid having to copy them
-# into this layer.
-RUN --mount=type=bind,source=package.json,target=package.json \
-  --mount=type=bind,source=package-lock.json,target=package-lock.json \
-  --mount=type=cache,target=/root/.npm \
-  npm ci --omit=dev
+RUN npm ci
 
-################################################################################
-# Create a stage for building the application.
-FROM deps as build
-
-# Download additional development dependencies before building, as some projects require
-# "devDependencies" to be installed to build. If you don't need this, remove this step.
-RUN --mount=type=bind,source=package.json,target=package.json \
-  --mount=type=bind,source=package-lock.json,target=package-lock.json \
-  --mount=type=cache,target=/root/.npm \
-  npm ci
-
-# Copy the rest of the source files into the image.
 COPY . .
-# Run the build script.
-RUN npm run build
 
-################################################################################
-# Create a new stage to run the application with minimal runtime dependencies
-# where the necessary files are copied from the build stage.
-FROM base as final
+# Build the application
+RUN npm run build
 
-# Use production node environment by default.
-ENV NODE_ENV production
+# Production stage
+FROM node:20
 
-# Run the application as a non-root user.
-USER node
+WORKDIR /app
 
-# Copy package.json so that package manager commands can be used.
-COPY package.json .
+COPY --from=builder /app/dist ./dist
+COPY package*.json ./
 
-# Copy the production dependencies from the deps stage and also
-# the built application from the build stage into the image.
-COPY --from=deps /app/node_modules ./node_modules
-COPY --from=build /app/dist ./dist
+RUN npm ci --omit=dev
 
+# We want jq and curl in the final image, but we don't need the support files
+RUN apt-get update && \
+	apt-get install -y jq curl tini && \
+	apt-get clean && \
+	rm -rf /usr/share/doc /usr/share/man /usr/share/zsh
 
-# Expose the port that the application listens on.
 EXPOSE 3000
 
-# Run the application.
-ENTRYPOINT ["/usr/bin/tini", "--", "node", "dist/apps/api/main.js"]
+ENTRYPOINT ["/usr/bin/tini", "--", "npm", "run", "start:api:prod"]
```

Comment:
Don't we want to prefer running node directly on built JS output, instead of using `npm run ...`?